MLP_Model/MLP_utils/exceptions.py

```diff
@@ -0,0 +1,26 @@
+# define python user-defined exceptions
+
+
+class TrainingValidationTestingSplitError(Exception):
+    """Exception occurs in the case that the training, validation, and testing splits to not add up to 1.0"""
+
+    pass
+
+
+class yDataTypeError(Exception):
+    """Exception occurs in the case that the y data does not have the correct data type
+    each model has a different y data type"""
```

Comment:
```suggestion
    (e.g., each model has a different y data type)"""
```
This description does not flow; it might be because I am reading it as a full sentence, but each line is meant to be a different sentence. I added a suggestion for what would make this clearer to me, but again it could be the way I am reading this docstring.